### ISSUE TITLE
Add criteria for existence of alternatives

### ISSUE THREAD

**Issue by KaKi87** [open-source-ideas/meta#14]
## Proposed change

Add a criteria to indicate whether a project has already been implemented in non-FLOSS projects.

## Motivation

Increase focus on projects that really have no functional alternative anywhere.

## Proposed implementation

Adding the following labels : `no alternative`, `premium`, `freemium`, `freeware`

**Comment by KOLANICH** [comment]
>`premium`, `freemium`, `freeware`

IMHO, better labels would be: `antifeature-legal` (restrictive non-free license), `antifeature-paid` (license may be free, but the owner de-facto requires a payment and there is no known way to bypass it), `antifeature-no-source` (the license may be free, but de-facto sources are unavailable), `antifeature-privacy-violation` (usage of all the existing alternatives causes violation of privacy of the persons using the software) and finally `antifeature-needs-another-maintainer` (the software may be free by itself, but the maintainer is uncooperative, i.e. refuses to even admit that the issues (i.e. any from the previous list) exist, or if he admits, refuses to fix them, and even to merge PRs fixing them, due to some reasons, i.e. some self-interests, such as commercial, so an another competing project may be needed to either push the maintainer to do what is needed or to just do what is needed and replace the original project).

**Comment by KaKi87** [comment]
Well, the terms I used should be the technically appropriate ones but whatever as long as it solves the issue. 😉

**Comment by FredrikAugust** [comment]
Good idea, but I personally don't think we need even more labels for this. A better solution in my opinion would be to add another section to the description such as:

### Do paid/free alternatives to this exist?

Or something along those lines.

Would love to hear your feedback on this @KaKi87 and @KOLANICH.

**Comment by KOLANICH** [comment]
And `Why are they not suitable`.

**Comment by FredrikAugust** [comment]
Because I think it would be more beneficial for whomever is reading the issue to see what alternatives exist, and perhaps draw inspiration from them, rather than just know that they exist.

**Comment by KaKi87** [comment]
Adding a section without adding tags won't help filtering and prioritizing.

**Comment by FredrikAugust** [comment]
No, I am aware. But I have to consider how much this will be used (for filtering) compared to the extra time it takes to create the labels, and assign them for each new project.

Furthermore, if people were to sort by these new labels, none of the old projects would come up.

**Comment by KaKi87** [comment]
I love offering my time for open source but time itself is not a renewable resource. So, I use this criteria for any project that anyone suggests me to do anyway. Not having a filter will mean I won't take time to review old suggestions but only the new ones, actually.
